.github/workflows/actions.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         include:
         # run rubocop against lowest supported ruby
         - os: ubuntu-latest
-          ruby: '2.5'
+          ruby: '2.7'
```

Comment:
I doubt this fixes the CI build error but just noticed it as I was trying to debug.

Reply:
This does seem to have fixed the error:

>Error when evaluating 'runs-on' for job 'build'. .github/workflows/actions.yml (Line: 14, Col: 14): Unexpected type of value '', expected type: OneOf.


But now there's an error saying the windows truffle ruby download was a 404.

https://github.com/ManageIQ/kubeclient/actions/runs/3740153807/jobs/6348284836

>Error: Error: Unexpected HTTP response: 404
    at D:\a\_actions\ruby\setup-ruby\v1\dist\index.js:8075:25
    at Generator.next (<anonymous>)
    at fulfilled (D:\a\_actions\ruby\setup-ruby\v1\dist\index.js:7985:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Not sure how to fix this one but doing a little research to see.

Reply:
That's ok, Truffleruby doesn't exist for windows.
I've tried to exclude it from the matrix but gave up for now.